src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct

```diff
@@ -8,11 +8,32 @@
       </Group>
     </Groups>
 
+    <Buttons>
+      <Button guid="ConnectedModeMenuCmdSet" id="ManageServerConnectionsCommand">
+        <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup"/>
```

Comment:
maybe place it into a different group, so it's distinct from the issue viz buttons?

Reply:
At the moment it's distinct thanks to the fact they _they're_ in a different group :)

Reply:
yeah ok, then it's all good